.tekton/operator-bundle-build.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stackrox/stackrox?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    # TODO(ROX-21073): re-enable for all PR branches
+    pipelinesascode.tekton.dev/on-cel-expression: (event == "push" && target_branch == "master") || (event == "pull_request" && (source_branch.contains("rhtap") || source_branch.contains("konflux") || source_branch.contains("appstudio")))
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: acs
+    appstudio.openshift.io/component: operator-bundle
+    pipelines.appstudio.openshift.io/type: build
+  name: operator-bundle-build
+  namespace: rh-acs-tenant
+spec:
+  params:
+  - name: dockerfile
+    value: operator/bundle.Dockerfile.konflux
```

Comment:
[nit] `.Dockerfile` is extension, at least some IDEs think this way. Please make it either `bundle.konflux.Dockerfile` or `konflux.bundle.Dockerfile`.